odoo/addons/test_access_rights/tests/test_feedback.py

```diff
@@ -183,29 +183,26 @@ def test_local(self):
             self.record.write({'val': 1})
         self.assertEqual(
             ctx.exception.args[0],
-            """Due to security restrictions, you are not allowed to modify 'Object For Test Access Right' (test_access_right.some_obj) records.
-
-Contact your administrator to request access if necessary.""")
+            """Uh-oh! Looks like you have stumled upon some top-secret Object For Test Access Rights.
```

Comment:
typo. should be stumbled I guess. Also, not putting quotes around object for test access rights makes the sentence quite confusing. I dont think using broken english like that gives a good impression